postman/Sila API v0.2 - JWT OAuth Edition.postman_collection.json

```diff
@@ -1,9 +1,10 @@
 {
 	"info": {
-		"_postman_id": "f9f3eba6-6c4a-40f2-a7dc-8980e29fd248",
+		"_postman_id": "6f73d634-4eb2-4d69-af0f-20a7a30b560f",
 		"name": "Sila API v0.2 - JWT OAuth Edition",
 		"description": "These endpoints are designed to work with our lightweight local proxy server. The local server signs and forwards requests to the main API host.\n\nAuthentication of requests made to the main API host is performed using ETH signatures. For more information, visit our docs at https://docs.silamoney.com/docs/process-overview.\n",
-		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "11558014"
```

Comment:
Apologies; in the last few days, we've moved the JWT/OAuth (which we're now calling "AuthToken") collection to a separate repo in order to begin to make this publicly available.

This "JWT OAuth Edition" collection is no longer being maintained, and should have been deleted from the repo, and this ticket (1633) updated to reflect that.  

I'll not bother requesting changes in this file, and we'll duplicate the work in the signer-server collection over to the new repo in-house.